site/content/docs/developing/domain.ja.md

```diff
@@ -84,9 +84,18 @@ Route53 以外のドメインを所有している場合や、コンプライア
 $ copilot env init --import-cert-arns arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
 ```
 
-Environment に Service をデプロイした後、その Environment に作成した ALB （Application Load Balancer） の DNS を、エイリアスのドメインがホストされている場所に A レコードとして追加してください。
+Service の Manifestにおいて、次のどちらかを行います。
```

Comment:
Could you add a space after the term Manifest?

Reply:
Yes. I added a space.